packages/cursorless-engine/src/processTargets/modifiers/constructScopeRangeTarget.ts

```diff
@@ -17,13 +17,20 @@ export function constructScopeRangeTarget(
   isReversed: boolean,
   scope1: TargetScope,
   scope2: TargetScope,
-): Target {
+): Target[] {
   if (scope1 === scope2) {
-    return scope1.getTarget(isReversed);
+    return scope1.getTargets(isReversed);
   }
 
-  const target1 = scope1.getTarget(isReversed);
-  const target2 = scope2.getTarget(isReversed);
+  const targets1 = scope1.getTargets(isReversed);
+  const targets2 = scope2.getTargets(isReversed);
+
+  if (targets1.length !== 1 || targets2.length !== 1) {
+    throw Error("Scope range targets must be single-target");
+  }
```

Comment:
wasn't sure what to do in this case.  seems like a corner case that will never happen.  But basically it's if user says "take tags air past bat", which doesn't really make any sense